frontend/static/js/component-editor.js

```diff
@@ -186,7 +186,7 @@ function createEditComponent() {
         "metrics": metrics
     }
 
-    if(document.getElementById('component-name').value=="") component_name_empty = true;
+    if(document.getElementById('component-name').value === "") component_name_empty = true;
```

Comment:
Gibt es einen Grund hier zu der dreifachen Variante überzugehen?

Reply:
Hey, ich finde es persönlich besser in js immer den === zu verwenden, weil dann die Abfrage eindeutiger ist. Gerade bei den unten genannten Beispielen auch in underem Fall könnte es sonst möglicherweise zu Problemen können.
  
  JavaScript has two sets of equality operators: === and !==, and their evil twins == and !=. The good ones work the way you would expect. If the two operands are of the same type and have the same value, then === produces true and !== produces false. The evil twins do the right thing when the operands are of the same type, but if they are of different types, they attempt to coerce the values. the rules by which they do that are complicated and unmemorable. These are some of the interesting cases:
    
    '' == '0'           // false
    0 == ''             // true
    0 == '0'            // true
    
    false == 'false'    // false
    false == '0'        // true
    
    false == undefined  // false
    false == null       // false
    null == undefined   // true
    
    ' \t\r\n ' == 0     // true
  
  von https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons